app/views/attach-pvc.html

```diff
@@ -76,7 +76,9 @@ <h2 class="text-center">No persistent volume claims.</h2>
                             </tbody>
                           </table>
                         </div>
-
+                        <div  ng-if="project && ('persistentvolumeclaims' | canI : 'create')">
+                          Or&nbsp;<a ng-href="project/{{project.metadata.name}}/create-pvc" >Create New Storage</a>
```

Comment:
usually in link form we don't use camel case on the actions, I would probably say `Or <a>create storage</a>`